umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-umh.md

```diff
@@ -132,6 +170,33 @@ Each component of the United Manufacturing Hub can be restored separately, in
 order to allow for more flexibility and to reduce the damage in case of a
 failure.
 
+### Copy kubeconfig file
+
+1. Show the content of kubeconfig file with the following command and copy the output:
```

Comment:
Specify that this command needs to be executed from the instance

Reply:
updated